modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -12,11 +12,11 @@ If you forward JSON logs to the default Elasticsearch instance managed by OpenSh
 
 You can use the following structure types in the `ClusterLogForwarder` CR to construct index names for the Elasticsearch log store:
 
-* `structuredTypeKey` (string, optional) is the name of a message field. The value of that field, if present, is used to construct the index name.
+* `structuredTypeKey` is the name of a message field. The value of that field, is used to construct the index name.
 ** `kubernetes.labels.<key>` is the Kubernetes pod label whose value is used to construct the index name.
 ** `openshift.labels.<key>` is the `pipeline.label.<key>` element in the `ClusterLogForwarder` CR whose value is used to construct the index name.
 ** `kubernetes.container_name` uses the container name to construct the index name.
-* `structuredTypeName`: (string, optional) If `structuredTypeKey` is not set or its key is not present, OpenShift Logging uses the value of `structuredTypeName` as the structured type. When you use both `structuredTypeKey` and `structuredTypeName` together,  `structuredTypeName` provides a fallback index name if the key in `structuredTypeKey` is missing from the JSON log data.
+* `structuredTypeName`: If `structuredTypeKey` is not set or its key is not present, OpenShift Logging uses the value of `structuredTypeName` as the structured type. When you use both `structuredTypeKey` and `structuredTypeName` together,  `structuredTypeName` provides a fallback index name if the key in `structuredTypeKey` is missing from the JSON log data.
```

Comment:
```suggestion
* `structuredTypeName`: If the `structuredTypeKey` field is not set or its key is not present, the {logging} uses the `structuredTypeName` value as the structured type. When you use both the `structuredTypeKey` field and the `structuredTypeName` field together, the `structuredTypeName` value provides a fallback index name if the key in the `structuredTypeKey` field is missing from the JSON log data.
```
Pretty sure you need to include something like field/value/key etc for things like this that are in backticks

Reply:
So `subsystem` was officially removed from the logging product name. The attribute hasn't been updated yet, in part because the grammar issue it creates makes it slightly more complex than it would otherwise be. I would strongly recommend refraining from adding instances of that attribute at the moment.